src/main/java/uk/gov/hmcts/reform/hmc/helper/GetHearingResponseMapper.java

```diff
@@ -167,52 +191,60 @@ private OrganisationDetails setOrganisationDetails(HearingPartyEntity hearingPar
+        individualDetails.setLastName(individualDetailEntity.getLastName());
+        individualDetails.setPreferredHearingChannel(individualDetailEntity.getChannelType());
+        individualDetails.setInterpreterLanguage(individualDetailEntity.getInterpreterLanguage());
+        if (null != hearingPartyEntity.getReasonableAdjustmentsEntity()
+                && !hearingPartyEntity.getReasonableAdjustmentsEntity().isEmpty()) {
+            individualDetails.setReasonableAdjustments(
+                    List.of(hearingPartyEntity.getReasonableAdjustmentsEntity().get(0)
+                            .getReasonableAdjustmentCode()));
+        }
+        individualDetails.setVulnerableFlag(individualDetailEntity.getVulnerableFlag());
+        individualDetails.setVulnerabilityDetails(individualDetailEntity.getVulnerabilityDetails());
+        if (null != hearingPartyEntity.getContactDetails()
+                && !hearingPartyEntity.getContactDetails().isEmpty()) {
+            if (hearingPartyEntity.getContactDetails().get(0).getContactDetails().contains("@")) {
+                individualDetails.setHearingChannelEmail(
+                        hearingPartyEntity.getContactDetails().get(0).getContactDetails());
+            } else {
+                individualDetails.setHearingChannelPhone(
+                        hearingPartyEntity.getContactDetails().get(0).getContactDetails());
+            }
+        }
+        RelatedParty relatedParty = new RelatedParty();
+        relatedParty.setRelatedPartyID(individualDetailEntity.getRelatedPartyID());
+        relatedParty.setRelationshipType(individualDetailEntity.getRelatedPartyRelationshipType());
+
+        individualDetails.setRelatedParties(List.of(relatedParty));
+        return individualDetails;
+    }
 
     private ArrayList<HearingResponse> setHearingResponse(HearingEntity hearingEntity) {
         ArrayList<HearingResponse> hearingResponses = new ArrayList<>();
         for (HearingResponseEntity hearingResponseEntity : hearingEntity.getHearingResponses()) {
             HearingResponse hearingResponse = new HearingResponse();
             hearingResponse.setListAssistTransactionID(
-                hearingResponseEntity.getHearingResponseId());
+                 hearingResponseEntity.getListAssistTransactionId());
             hearingResponse.setReceivedDateTime(hearingResponseEntity.getRequestTimeStamp());
             hearingResponse.setResponseVersion(hearingResponseEntity.getHearingResponseId());
-            hearingResponse.setLaCaseStatus(hearingResponseEntity.getListingCaseStatus());
-            hearingResponse.setListingStatus(hearingResponseEntity.getListingStatus());
+            hearingResponse.setLaCaseStatus(ListAssistCaseStatus.getLabel(
+                    hearingResponseEntity.getListingCaseStatus()));
+            hearingResponse.setListingStatus(ListingStatus.getLabel(hearingResponseEntity.getListingStatus()));
```

Comment:
hearingCancellationReason value is not populated. Should be retrieved from hearingResponse.cancellationReasonType

Reply:
Code updated. Thanks.